mux.go

```diff
@@ -164,13 +164,18 @@ func (r *Router) GetRoute(name string) *Route {
 // StrictSlash defines the trailing slash behavior for new routes. The initial
 // value is false.
 //
-// When true, if the route path is "/path/", accessing "/path" will redirect
+// When true, if the route path is "/path/", accessing "/path" will perform a  redirect
```

Comment:
Need to fix. Can't do that in GitHub.

Reply:
??

Reply:
Introduced a space and can’t edit my branch via the web UI (not at my laptop) 😉